sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -209,8 +213,11 @@ class Version(Base):
     id = Column(Integer, Identity(start=1, increment=1), primary_key=True)
     instrument = Column(INSTRUMENTS, nullable=False)
     data_level = Column(DATA_LEVELS, nullable=False)
+    # TODO: determine cap for strings based on what software version
```

Comment:
I think we would be removing software version now? All of that is wrapped up in the single version scheme now, whereas before it was `software-data` and we could parse them out.

Reply:
can we remove this in future PR when we incorporate version changes? This will remind us to decide on we should track software version.

Reply:
OK, makes sense then!